LoopKitUI/Views/GuardrailWarning.swift

```diff
@@ -79,7 +79,7 @@ public struct GuardrailWarning: View {
                 }
             } else {
                 // ANNA TODO: figure out this one
-                return Text("Some of the values you have entered are outside of what Tidepool typically recommends for most people.")
+                return Text(LocalizedString("Some of the values you have entered are outside of what is typically recommended for most people.", comment: "Caption for guardrail warning when more than one threshold is crossed"))
```

Comment:
This will need to be approved by Melissa Lee.

Reply:
We have discussed this several times. Tidepool specific text in code common to Tidepool Loop and DIY needs to wait until we have the facility for Tidepool Loop string substitution, or it will block merges.